ballerina/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 distribution = "2201.10.0"
 org = "ballerinax"
 name = "salesforce"
-version = "8.1.0"
```

Comment:
The change is more likely a breaking change (as we are adding new fields to a closed record), hence better to check the possibility of releasing a major version or a minor-version with a new distribution update version.